Firebase.podspec

```diff
@@ -69,6 +69,11 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     ss.dependency 'FirebaseAuth', '~> 6.4.1'
   end
 
+  s.subspec 'Crashlytics' do |ss|
+    ss.dependency 'Firebase/CoreOnly'
+    ss.ios.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
```

Comment:
Any reason that tvos and macos are excluded?

Reply:
Let me add them